music21/romanText/writeRoman.py

```diff
@@ -174,8 +173,10 @@ def __init__(self,
     def _makeContainer(self,
                        objs: list):
         '''
-        Makes a placeholder container for the usual cases where this class is called on
-        generic- or non-stream object.
+        Makes a placeholder container for the unusual cases where this class is called on
+        generic- or non-stream object as opposed to
+        a :class:~music21.stream.Score, :class:~music21.stream.Part,
```

Comment:
Everything after the `:class:` needs to be wrapped in backticks: `

(When I (often) need a refresher, I just pop `:class:` into my search bar in my code editor to pull up examples.)

Reply:
Thanks! Does the Opus handling look alright to you?

Reply:
I'll take a look this week. At a glance looks good!